lib/recurly.js

```diff
@@ -296,9 +334,10 @@ export class Recurly extends Emitter {
       this.emit('error', ...args);
     });
 
-    if (this.bus) this.bus.destroy();
-    this.bus = new Bus({ api: this.config.api });
-    this.bus.add(this);
+    if (!this.bus) {
```

Comment:
Why the change here? To prevent an unnecessary `this.bus.destroy()` call?

Reply:
Coming back to this after chatting some in-person. This commit message goes into this change in detail: https://github.com/recurly/recurly-js/commit/8dacd36fb5a5ab5fb135b306374ea6cdb739494c